README.MD

```diff
@@ -76,6 +77,12 @@ programmer among salesmen etc?
 - How is the project automated? 
 - How often are there releases?
 - Do they have continuous integration?
+- Is your infrastructure setup under version control / available as code?
+- Tell me about your technical principles or vision.
+- How do you prepare for disaster recovery?
+- How do you track bugs?
+- How do you integrate and deploy changes? Is it CI/CD?
```

Comment:
There is point `Do they have continuous integration?`
This one is better, but please remove previous one

Reply:
Are you saying to remove "Do they have continuous integration?"